instance/tests/models/test_instance.py

```diff
@@ -137,9 +137,10 @@ def test_github_admin_username_list_default(self):
         By default, no admin should be configured
         """
         instance = OpenEdXInstanceFactory()
+        instance.reset_ansible_settings()
```

Comment:
@antoviaque I actually think freezing the configuration at the time the instance is provisioned is the right thing to do.

The manager create method is on the `GitHubInstanceQuerySet` class, which is the wrong place to add anything Ansible-specific.  I would have to create a mixin for the query set that is combined with `GitHubInstanceQuerySet` to take care of this.

The `on_pre_save()` hook is also the wrong place, since we don't want to reset the configuration every time the instance is saved.  The configuration can only possibly change at the time the instance is provisioned, and before it's provisioned, it doesn't really have an Ansible config.

The only place where you have to think of calling the method is in the tests.  I could add a hook to the factory class to avoid that problem.